sdk/communication/communication-phone-numbers/src/phoneNumbersClient.ts

```diff
@@ -116,22 +115,22 @@ export class PhoneNumbersClient {
   }
 
   /**
-   * Gets the details of an acquired phone number. Includes phone number, cost, country code, etc.
+   * Gets the details of a purchased phone number. Includes phone number, cost, country code, etc.
    *
    * @param {string} phoneNumber The E.164 formatted phone number being fetched. The leading plus can be either + or encoded as %2B.
-   * @param {GetPhoneNumberOptions} options Additional request options.
+   * @param {GetPurchasedPhoneNumberOptions} options Additional request options.
```

Comment:
Can you run the linter? I think it will complain about missing dashes.
```suggestion
   * @param {GetPurchasedPhoneNumberOptions} options - Additional request options.
```